data-sources/github-integration/config.yml

```diff
@@ -0,0 +1,13 @@
+id: github-integration
+displayName: GitHub Integration
+description: |
+  Seamlessly adopt New Relic Service catalog with the GitHub Integration.
+icon: logo.svg
+install:
+  primary:
+    link:
+      url: https://www.newrelic.com
```

Comment:
Please provide the relative URL for the GitHub integration documentation page.

Reply:
Updated!